core/ledger/kvledger/txmgmt/statedb/statecouchdb/couchdoc_conv.go

```diff
@@ -184,6 +184,10 @@ type couchSavepointData struct {
 	TxNum    uint64 `json:"TxNum"`
 }
 
+type channelMetadata struct {
+	NamespaceDBsInfo []*namespaceDBInfo `json:"NamespaceDBsInfo"`
+}
+
```

Comment:
```suggestion
type channelMetadata struct {
      ChannelName      string             `json:"ChannelName"`
	NamespaceDBsInfo []*namespaceDBInfo `json:"NamespaceDBsInfo"`
}
```
`ChannelName` filed helps in indireclty storing the channelName associated with a metadataDB. Otherwise, we can store the name explicty as well.